packages/playground/src/app.jsx

```diff
@@ -480,6 +480,8 @@ class Playground extends Component {
 
     return (
       <div className='container-fluid'>
+
+        {JSON.stringify(liveSettings)}
```

Comment:
I'm not sure you meant to leave this in here? It seems odd to render them as a string